### PR TITLE
update bundled plugins for server version 5.24

### PR DESCRIPTION
**Pull request by jfrerich**
#### Summary
Upgrade bundled plugins per https://mattermost.atlassian.net/browse/MM-25318

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-25318

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,16 +85,16 @@ TESTS=.
...
 # Plugins Packages
-PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.0
-PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.1.2
+PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.1
+PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-nps-v1.0.4
-PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.0.2
-PLUGIN_PACKAGES += mattermost-plugin-github-v0.11.0
+PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.2.0
+PLUGIN_PACKAGES += mattermost-plugin-github-v0.14.0
```
Comment (hanzei) [inline Makefile:92]: This should be `v1.0.0`

File: Makefile [hunk 2]
```diff
@@ -85,16 +85,16 @@ TESTS=.
...
 PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.1.1
```
Comment (hanzei) [inline Makefile:93]: Maybe ship `v1.2.0`?

File: Makefile (hunk 1 above)
Comment (jfrerich) [reply Makefile:92]: Sorry, I took the versions directly from the Jira ticket.

File: Makefile (hunk 2 above)
Comment (jfrerich) [reply Makefile:93]: I don't see why not.

File: Makefile [hunk 3]
```diff
@@ -85,16 +85,16 @@ TESTS=.
 TE_PACKAGES=$(shell $(GO) list ./...)
 
 # Plugins Packages
-PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.0
-PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.1.2
+PLUGIN_PACKAGES?=mattermost-plugin-zoom-v1.3.1
+PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.2.0
```
Comment (hanzei) [inline Makefile:89]: Given that autolink `v1.2.0` contains a breaking change that we want to fix, I'm 3/5 that we should not force an upgrade.

Comment (jfrerich) [reply Makefile]: Thanks @hanzei.  I reverted to the previous version.

**Comment by jfrerich** [comment]
@aaronrothschild and @DHaussermann.  Can you provide feedback to @levb's comment.  We can leave jira as 2.3 and users can still choose to upgrade via marketplace.

**Comment by aaronrothschild** [comment]
> @aaronrothschild and @DHaussermann. Can you provide feedback to @levb's comment. We can leave jira as 2.3 and users can still choose to upgrade via marketplace.

Agree with Lev, let's play it safe at this point.

**Comment by jfrerich** [comment]
Thanks @aaronrothschild.  Jira will remain at `v2.3`

File: Makefile [hunk 4]
```diff
@@ -85,16 +85,16 @@ TESTS=.
...
-PLUGIN_PACKAGES += mattermost-plugin-github-v0.11.0
-PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.1.1
+PLUGIN_PACKAGES += mattermost-plugin-custom-attributes-v1.2.0
+PLUGIN_PACKAGES += mattermost-plugin-github-v1.0.0
+PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-aws-SNS-v1.0.2
 PLUGIN_PACKAGES += mattermost-plugin-antivirus-v0.1.2
-PLUGIN_PACKAGES += mattermost-plugin-jira-v2.3.2
-PLUGIN_PACKAGES += mattermost-plugin-gitlab-v1.0.1
+PLUGIN_PACKAGES += mattermost-plugin-jira-v2.3.0
```
Comment (levb) [inline Makefile:96]: Wait, why did we go from 2.3.2 -> 2.3.0?

Comment (jfrerich) [reply Makefile]: Fixed

**Comment by jfrerich** [comment]
@hanzei, in the Jira ticket, gitlab was requested to upgrade to `v1.1.0`.  This is causing the build to fail.  Any ideas on how to solve?  That release is old and perhaps the release file names is the issue?
image
versus github
image

**Comment by hanzei** [comment]
@ali-farooq0 did some changes a couple of month ago that required a re-build of every release that should be pre-packaged. (https://github.com/mattermost/matterbuild/pull/17) Re-building the gitlab releases fixed the issue.

**Comment by ali-farooq0** [comment]
@jfrerich Could be a few things -- do you know what the error was?

**Comment by jfrerich** [comment]
@ali-farooq0, I don't recall the exact error.  I also see that the error is gone now, even though I haven't added any new commits.   Weird, but all looks good now.

**Comment by DHaussermann** [comment]
@hanzei, I have a related question. While testing with steps above, I noticed that several plugins that should have a marketplace icon do not have one when remote marketplace is disabled. (AWS, GitLab, Jenkins, Jira and Welcome Bot)

Do you know if this is expected? Perhaps I should create a marketplace issue if we're not sure what's causing this?

**Comment by jfrerich** [comment]
@hanzei and @levb, please confirm the latest changes!

**Comment by hanzei** [comment]
@DHaussermann Yes that is expected. There a two sources for icon in the Marketplace modal: The Marketplace itself or the plugin bundle. For the plugin you linked above, there is no icon in the plugin bundle and if the  Marketplace is disabled, no icon will be shown. Once we bump the versions on these plugin, they will also have an icon if the Marketplace is disabled.

Please note that Welcome Bot doesn't have any icon.

**Comment by amyblais** [comment]
@jfrerich this hasn't been cherry picked yet

**Comment by jfrerich** [comment]
Hi @amyblais!  correct, I am doing the cherry-pick right now!

**Comment by jfrerich** [comment]
... to `release-5.24`

**Pull request merged**